libs/blocks/global-navigation/global-navigation.css

```diff
@@ -316,6 +317,22 @@ header.global-navigation {
 .feds-breadcrumbs li {
   display: flex;
   align-items: center;
+  flex-shrink: 0;
+}
+
+/* Hide all breadcrumbs except the first and last two */
+.feds-breadcrumbs li:nth-last-child(n+3):not(:first-child) {
+  display: none;
+}
+
+/* If first breadcrumb is not third to last, add ellipsis after it */
+.feds-breadcrumbs li:first-child:not(:nth-last-child(-n+3)):after {
+  content: '/\2003...';
```

Comment:
In Firefox and Safari the spacing doesn't come out quite right with this. What about breaking out the "/" and the "...", with the "/" going in :before and "..." in :after so you can control the spacing more consistently? 
<img width="307" alt="image" src="https://github.com/adobecom/milo/assets/7562106/9d3f52c0-e204-40d7-994f-e3beccf05587">

Reply:
Just a small add, inconsistent spacing replicate for Safari and FF on MAC desktop resizing browser.  
On real devices (Android and Iphone ) works properly for all browsers. 
<img width="394" alt="Safari_IOS" src="https://github.com/adobecom/milo/assets/15346184/3e6bf5bf-2fad-4ad9-a632-8001beb2e8a5">
![FF_android](https://github.com/adobecom/milo/assets/15346184/fa0d313f-5525-4147-8e85-17b9a32e7484)
![FF_IOS](https://github.com/adobecom/milo/assets/15346184/2ba5b810-3572-4ea6-91ee-42a306c41eea)